gbus/bus.go

```diff
@@ -29,6 +29,7 @@ type DefaultBus struct {
 	PrefetchCount  uint
 	AmqpConnStr    string
 	amqpConn       *amqp.Connection
```

Comment:
let's rename this variable to ingressConn

Reply:
all fixed